ksp_plugin/vessel.cpp

```diff
@@ -577,9 +577,12 @@ void Vessel::StartPrognosticatorIfNeeded() {
 }
 
 Status Vessel::RepeatedlyFlowPrognostication() {
+  std::chrono::steady_clock::time_point wakeup_time =
+      std::chrono::steady_clock::now();
   for (;;) {
+    std::this_thread::sleep_until(wakeup_time);
     // No point in going faster than 50 Hz.
-    std::chrono::steady_clock::time_point const wakeup_time =
+    wakeup_time =
         std::chrono::steady_clock::now() + std::chrono::milliseconds(20);
```

Comment:
It would probably be better to write this:
```
wakeup_time += std::chrono::milliseconds(20);
```
This would save reading the clock each time, and it would avoid drifts in the wake-up time of the loop (if the thread is suspended between waking-up and reading the clock).

Reply:
Done (using eggrobin@'s suggestion).